src/Maker/DddMaker.php

```diff
@@ -25,6 +27,7 @@
  */
 abstract class DddMaker extends AbstractMaker
 {
+    protected const DOMAIN_NAME_OPTION_KEY = "domain-name";
```

Comment:
Missing spacing between const and constructor/fields.

```suggestion
    protected const DOMAIN_NAME_OPTION_KEY = "domain-name";

```